### DISCUSSION TITLE
Helm Repo

### DISCUSSION THREAD

**Discussion by alaabouz** [magma/magma#14744] [Q&A] [answered]
Hello,
I am sorry if this question have been asked previously, i am new to k8s and i am stucked in the setup of orc8r_helm_repo
do you know any public repo for that or i have to build my own charts ? not sure if i am asking in the good way but if someone can guide me it will be really appreciated !
Thanks in advance.

**Comment by sebathomas** [comment]
Hi,
the helm charts can be found at https://linuxfoundation.jfrog.io/ui/native/magma-helm

See https://magma.github.io/magma/docs/orc8r/dev_minikube#deploy-on-minikube for how to deploy on minikube or https://magma.github.io/magma/docs/orc8r/deploy_intro for AWS.

**Reply by alaabouz** [reply]
Thank you ! do you know where can i find the public docker images ? as i am having a lot of errors when running the build.py script

**Reply by sebathomas** [reply]
https://linuxfoundation.jfrog.io/ui/repos/tree/General/magma-docker

(You may need to create a linuxfoundation account to use the UI.)

**Reply by alaabouz** [reply]
Thanks again ! just last question what is the best practice to use the repo's you shared in the ansible_vars file do i copy it to another public github repo then i use it or there is a method to access it directly from the server using a username/pwd ?

**Reply by sebathomas** [reply]
You can just get them from URLs like `linuxfoundation.jfrog.io/magma-docker/magmalte` for example, no login necessary. The account is only need to access the UI.

See this helm values file for more examples: https://github.com/magma/magma/blob/master/orc8r/cloud/helm/orc8r/values.yaml

**Reply by alaabouz** [reply]
Thank you soo much ! i really progressed a lot in the installation, i deceded to go with minikube installation procedure now i am almost done, all the pods are up 
kubectl --namespace orc8r get pods
NAME                                            READY   STATUS    RESTARTS   AGE
nms-magmalte-6f4859fd65-78hgf                   1/1     Running   0          30m
nms-nginx-proxy-5bc7cc4d9d-tpt25                1/1     Running   0          30m
orc8r-accessd-66d8dd8f44-kf2x9                  1/1     Running   0          30m
orc8r-alertmanager-6b7d5fc7d8-gnhbd             1/1     Running   0          30m
orc8r-alertmanager-configurer-f77bf587f-p7ngw   1/1     Running   0          30m
orc8r-analytics-5c6ff586b-2d548                 1/1     Running   0          30m
orc8r-bootstrapper-565d47f469-jp4zj             1/1     Running   0          30m
orc8r-certifier-64c8d4449f-m7plr                1/1     Running   0          30m
orc8r-configurator-7fcd448747-p8v9d             1/1     Running   0          30m
orc8r-ctraced-5f45fd76f4-z8q4g                  1/1     Running   0          30m
orc8r-device-7464b7666f-k22ls                   1/1     Running   0          30m
orc8r-directoryd-59898cf444-t4bn8               1/1     Running   0          30m
orc8r-dispatcher-6fb8bcb5c6-26dw2               1/1     Running   0          30m
orc8r-eventd-c6cdb9776-tv9s9                    1/1     Running   0          30m
orc8r-metricsd-5bbf8f94b7-mvzzp                 1/1     Running   0          30m
orc8r-nginx-6d9d467878-wmxqt                    1/1     Running   0          30m
orc8r-obsidian-57d74b46c5-jb7hg                 1/1     Running   0          30m
orc8r-orc8r-worker-7d4cdbd77-lpwkn              1/1     Running   0          30m
orc8r-orchestrator-8664564cf8-l44xs             1/1     Running   0          30m
orc8r-prometheus-57475f847c-mj82s               1/1     Running   0          30m
orc8r-prometheus-cache-9d8d9dcfc-nrvnf          1/1     Running   0          30m
orc8r-prometheus-configurer-b66cd766b-bwm52     1/1     Running   0          30m
orc8r-service-registry-8674bbbc4c-lgjmr         1/1     Running   0          30m
orc8r-state-56755bfc6b-tgtnh                    1/1     Running   0          30m
orc8r-streamer-fb6bd99b7-xdbjl                  1/1     Running   0          30m
orc8r-tenants-597fcd6b55-gnclr                  1/1     Running   0          30m
orc8r-user-grafana-6655cf5bb5-ztqxh             1/1     Running   0          30m
postgresql-0                                    1/1     Running   0          46m

i added magma-test.localhost to my PC hosts file with the IP, but when trying to access the nms i am getting "Unable to connect"
any advise ?
Thanks

**Reply by sebathomas** [reply, marked as answer]
I'm not sure if you need to modify your hosts file to access nms but I haven't used minikube in a while. Does the access to the orc8r swagger api work? https://magma.github.io/magma/docs/orc8r/dev_minikube#access-orc8r

(If you're only trying to play around with orc8r/nms, it might be easier to set them up in a docker-compose dev environment following https://magma.github.io/magma/docs/next/basics/quick_start_guide.)

**Reply by alaabouz** [reply]
thanks yes the api work i am getting the "hello" didn't work without using sudo do you think it's related ?

**Reply by sebathomas** [reply]
Sometimes there is a bug that assigns the certificates to root, that might cause problems. You can take a look at the files in `$MAGMA_ROOT/.cache/test_certs` and chown them to your user if they are owned by root.

**Reply by alaabouz** [reply]
unfortunately that didn't solve the issue i am still getting
This site can’t be reachedmagma-test.localhost refused to connect.

 i will restart the installatoin using docker as you recommended.
thanks for all support

**Reply by alaabouz** [reply]
Hello,
Happy New Year ! finally everything is working with Docker compose thanks for all support provided !
Alaa.
